packages/svelte/src/internal/client/reactivity/effects.js

```diff
@@ -249,6 +262,19 @@ export function render_effect(fn) {
 	return create_effect(RENDER_EFFECT, fn, true);
 }
 
+/**
+ * @param {() => void | (() => void)} fn
+ * @returns {import('#client').Effect}
+ */
+export function template_effect(fn) {
+	if (DEV) {
+		define_property(fn, 'name', {
+			value: 'Svelte template effect'
```

Comment:
I think we should avoid 'effect' terminology here, as people don't necessarily think of these as effects

```suggestion
			value: '{expression}'
```